scripts/dxcompiler_copy/dxapp.json

```diff
@@ -109,7 +116,7 @@
     }
   },
   "details": {
-    "whatsNew": "* 0.0.5: updated azure instances to v2 for OFH region\n * 0.0.3: Added new London OFH region\n * 0.0.2: Added new London region."
+    "whatsNew": "* 0.0.6: Added new Bahrain AWS region aws:me-south-1 \n * 0.0.5: updated azure instances to v2 for OFH region\n * 0.0.3: Added new London OFH region\n * 0.0.2: Added new London region."
```

Comment:
minor: extra space

Reply:
hopefully in [439e46e](https://github.com/dnanexus/dxCompiler/pull/481/commits/439e46e14198b41b361e2ed325662acef845179c)